### ISSUE TITLE
In Grafana dashboard the "CPU" metric is sometimes bigger than "pkg" metric

### ISSUE THREAD

**Issue by Feelas** [sustainable-computing-io/kepler#125]
**Describe the bug**
CPU metric from the same timeframe is sometimes bigger than pkg metric on a platform where (as discussed in #120) Core is derived from Pkg. On the presented screenshot, the green metric is pkg (labeled as "Total") and the yellow one is Core - there are numerous periods when Core is higher than pkg, despite pkg is derived from Core. 

_This is possibly an artifact of Grafana queries & processing done when fetching the data, as querying same data directly from Prometheus through a different UI shows no cuts between two graphs._

**To Reproduce**
No specific trigger scenario, observe workload metrics in Grafana.

**Expected behavior**
'Core' metric is not higher than 'Pkg' in Grafana observable metrics. 

**Screenshots**
Total being lower than core in Grafana in the specific time period - there are cuts between both graphs
![Zrzut ekranu (104)](https://user-images.githubusercontent.com/43238042/187922401-a245808b-2018-4a06-aca7-6395b6849885.png)
Pkg not crossing with core on different graph (fetched through OpenShift -> Observe -> Metrics query): Blue is curr_energy_in_core, yellow is curr_energy_in_pkg.
![prometheus-core-pkg-export](https://user-images.githubusercontent.com/43238042/187922786-1c32bf08-df36-4087-a2e0-ebff6d41839c.JPG)

**Desktop**
- OpenShift 4.11 cluster

**Comment by rootfs** [comment]
@Feelas thanks for finding this! Is the pod level metric or node level?

cc @sunya-ch

**Comment by Feelas** [comment]
@rootfs This is pod level, I didn't take a look at node level.

**Comment by sunya-ch** [comment]
Is this the query you are using? 

[          "expr": "rate(pod_curr_energy_in_core_millijoule{pod_namespace=\"$namespace\", pod_name=\"$pod\"}[1m])/3",
](https://github.com/sustainable-computing-io/kepler/blob/e285f319878cc45fb7cc2a59242b7438d4f3acad/grafana-dashboards/Kepler-Exporter.json#L224)

If we are going to apply rate function, I think we should apply it to the total value instead of curr value. 
The following two queries should be equivalent.
```
rate(pod_total_energy_in_core_millijoule{pod_namespace=\"$namespace\", pod_name=\"$pod\"}[1m])
```
or 
```
pod_curr_energy_in_core_millijoule{pod_namespace=\"$namespace\", pod_name=\"$pod\"}/3
```

**Comment by Feelas** [comment]
@sunya-ch might be something obvious, but why exactly is there division by three applied to pod_curr_energy_in_core? I've seen this on every dashboard present in the repository, but I don't quite understand where the 1/3 scaling comes from.

**Comment by sunya-ch** [comment]
3s is the kepler default ticker period. Current value is the gauge value that includes value only in that period. So, we divide it by 3 to average for 1s. 
For total value, it is accumulated value. Applying rate/irate function will compute per-second rate from accumulated counter. So, there is no need to divide it by 3 again.

**Comment by Feelas** [comment]
I think that this might have been caused by the "uncore metric bug" described in #132 and fixed by #136. I didn't notice this issue after moving to image patched with PR 136.

Despite that, due to what @sunya-ch mentioned w/ regards to using rate on total metrics or curr metrics, a dashboard metric review would be a good next step to take, I think.

**Comment by rootfs** [comment]
@Feelas cool! PR is welcome!
